loops_labs.js

```diff
@@ -0,0 +1,231 @@
+# Loops Lab
+
+To complete this lab, do the following:
+
+1. Fork this repo
+2. Clone **your** forked repo
+3. Create a file called `loops_lab.js` inside your cloned repo
+4. Copy the description of the exercise as a comment
+5. Underneath the commented description write your code.
+6. Ensure your solutions work properly!
+7. Open PR against this repo.
+
+___
+
+## Exercises
+
+1. Write a `while` loop and a `for` loop that takes the variable
```

Comment:
Good job overall with lab. I can see good demonstration of for and while loops, and their implementations. Keep up the good work and keep practicing as getting the hang of loops is super important for future lessons as well.